feature-libs/asm/core/facade/asm.service.ts

```diff
@@ -53,31 +53,6 @@ export class AsmService {
     );
   }
 
-  /**
-   * Search for customers in a customer list
-   */
-  customerListCustomersSearch(options: CustomerSearchOptions): void {
-    this.store.dispatch(new AsmActions.CustomerListCustomersSearch(options));
-  }
-
-  /**
-   * Returns the customer search result data for a customer list
-   */
-  getCustomerListCustomersSearchResults(): Observable<CustomerSearchPage> {
-    return this.store.pipe(
-      select(AsmSelectors.getCustomerListCustomersSearchResults)
-    );
-  }
-
-  /**
-   * Returns the customer list customers search result loading status.
-   */
-  getCustomerListCustomersSearchResultsLoading(): Observable<boolean> {
-    return this.store.pipe(
-      select(AsmSelectors.getCustomerListCustomersSearchResultsLoading)
-    );
-  }
-
   /**
    * Returns the customer list customers search result error status.
```

Comment:
Hi @ercultimate , It looks like the method should also move to AsmCustomerListFacade? Am I right?

Reply:
Okay, I have changed it